list1.py

```diff
@@ -54,10 +64,11 @@ def front_x(words):
 #   [(2, 2), (1, 3), (3, 4, 5), (1, 7)]
 # Hint: use a custom key= function to extract the last element form each tuple.
 
+def last(t):
+    return t[-1]
 
 def sort_last(tuples):
-    # your code here
-    return
+    return sorted(tuples, key=last)
```

Comment:
Nice use of a sort function here. You can actually do this all on one line using a lambda function instead of creating another function!
`sorted(tuples, key=lambda t:t[-1])`
https://www.w3schools.com/python/python_lambda.asp